### PR TITLE
Add support for composer

### PR DESCRIPTION
**Pull request by killua99**
Hi,

I'll like to provide a support for composer package system https://getcomposer.org/ because there is a few modules or plugins for PHP framework that require this library to be able to work with it. Could be much easier for us to fetch this library via composer.

For example, this Drupal module require this library to be able to work with it. https://www.drupal.org/project/file_browser

Will be nice to have this.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "desandro/imagesloaded",
```
Comment (desandro) [inline composer.json]: Should be desandro/masonry

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "description": ":love_hotel: Cascading grid layout library http://masonry.desandro.com",
```
Comment (desandro) [inline composer.json]: Please change to just `Cascading grid layout library`

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "type": "library",
+  "keywords": ["javascript", "library", "grid", "browser", "dom", "layout", "outlayer"],
+  "homepage": "https://github.com/desandro/masonry",
```
Comment (desandro) [inline composer.json]: Please change to `http://masonry.desandro.com`

**Comment by desandro** [comment]
Thanks for the contribution. Can you make a couple fixes?

**Comment by killua99** [comment]
Yes I'll! Awesome! Thanks!

File: composer.json (hunk 3 above)
Comment (killua99) [inline composer.json]: But the homepage shouldn't be the github repo ? Anyway the description has the webpage.

**Comment by killua99** [comment]
The last thing should be register them at https://packagist.org/login/ but at the moment this is a great support for us! thanks!

**Comment by desandro** [comment]
[desandro/masonry registered on packagist.org with auto-updates](https://packagist.org/packages/desandro/masonry). I'm not familiar with Packagist, so let me know if there's something that needs to be fixed there

**Pull request merged**